packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4752,7 +4798,16 @@ export class ContainerRuntime
 				this.channelCollection.rollback(type, contents, localOpMetadata);
 				break;
 			}
-			case ContainerMessageType.GC:
+			case ContainerMessageType.GC: {
+				// Just drop it, but log an error, this is not expected and not ideal, but not critical failure either.
+				// Currently the only expected type here is TombstoneLoaded, which will have been preceded by one of these events as well:
+				// GC_Tombstone_DataStore_Requested, GC_Tombstone_SubDataStore_Requested, GC_Tombstone_Blob_Requested
+				this.mc.logger.sendErrorEvent({
```

Comment:
Is it possible to log more details here such as the id of the object, pkg, etc? That will help debugging if and when we see this log. As of now, debugging this will be very hard.

Reply:
I think it'll be ok because there will be a corresponding `GC_Tombstone_DataStore_Requested` (or other object) event with more info